holoviews/element/chart.py

```diff
@@ -281,3 +281,14 @@ def stack(cls, areas):
                                     new_type=Area)
             baseline = baseline + y
         return stacked
+
+class Segments(Geometry):
```

Comment:
Let's move this definition to ``elements/geom.py``

Reply:
Agreed.